nixos/modules/services/system/systemd-lock-handler/default.nix

```
@@ -0,0 +1,33 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.services.systemd-lock-handler;
+in {
+  options.services.systemd-lock-handler = {
+    enable = mkEnableOption (lib.mdDoc "systemd-lock-handler");
+    package = mkOption {
+      default = pkgs.systemd-lock-handler;
+      defaultText = literalExpression "pkgs.systemd-lock-handler";
+      type = types.package;
+      description = lib.mdDoc "systemd-lock-handler package to use.";
+    };
```

There seems to be a helper, like `mkEnableOption`, for this:

```suggestion
    package = mkPackageOption pkgs "systemd-lock-handler" { };
```